e2e/tests/nightly/admin/lock-workspace.spec.ts

```diff
@@ -124,6 +125,11 @@ describe('Workspace Admin lock-workspace', () => {
     await workspaceEdit.waitForLoad();
     await workspaceEdit.clickCancelButton();
     // cleanup
-    await dataPage.deleteWorkspace();
+    try {
+      await dataPage.deleteWorkspace();
```

Comment:
Why cant we delete the workspace in after block?

Reply:
Sometimes workspace fail to delete due to various issues, like Terra issue or some APP test uses that workspace but not deleting that.

Reply:
My only concern is if we run it on local, it leaves behind workspaces which could be distracting

Reply:
Given the above conversation, this wants a comment or Joel's suggestion. Something like:
```javascript
// Attempt to clean up here, reducing the load on the nightly cron job.
```